pkg/quota/interfaces.go

```diff
@@ -64,3 +64,19 @@ type Registry interface {
 	// Evaluators returns the set Evaluator objects registered to a groupKind
 	Evaluators() map[unversioned.GroupKind]Evaluator
 }
+
+// UnionRegistry combines multiple registries.  Order matters because first registry to claim a GroupKind
+// is the "winner"
+type UnionRegistry []Registry
+
+func (r UnionRegistry) Evaluators() map[unversioned.GroupKind]Evaluator {
+	ret := map[unversioned.GroupKind]Evaluator{}
+
+	for i := len(r) - 1; i >= 0; i-- {
+		for k, v := range r[i].Evaluators() {
```

Comment:
> what is the behavior intended here when there are more than 1 evaluators?

More than one registry will end up with "first registry claim a GroupKind wins".  More than one evaluator can't collide on GroupKind since its in a map, so the order won't matter.